Makefile

```diff
@@ -161,6 +162,8 @@ gendoc: $(DOCDIR)
 	# added copying of images and renaming of TEMP.md
 	cp $(SRC)/docs/*md $(DOCDIR) ; \
 	cp -r $(SRC)/docs/images $(DOCDIR) ; \
+	$(RUN) gen-prefix-map -o project/prefixmap/nmdc-prefix-map.json $(SOURCE_SCHEMA_PATH) ; \
```

Comment:
Just a thought for an improvement -- should we have `gen-prefix-map` also be able to return a Markdown file with the same results like the JSON would have, but instead of key-value pairs we would have a table of the prefixes?

That way we can have mkdocs convert the Markdown file into HTML and have it be shown as part of our web doc pages.

Definitely not a priority, but a nice to-have I would say.

Reply:
Maybe add an option `--format` have it be restricted to `[json|md]`

Reply:
@sujaypatil96 - I can see how that would be a nice doc; I do think the purpose of these is primarily programmatic.